### ISSUE TITLE
Spearhead: Fatal error when WooCommerce is active. 

### ISSUE THREAD

**Issue by kjellr** [Automattic/themes#3005]
## Steps to replicate

Install + activate WooCommerce on a test site. 
Visit the front end, and note this error appears: 

```
Fatal error
: Uncaught Error: Call to undefined function spearhead_cart_link() in .../wp-content/themes/dotcom/spearhead/header.php:106 Stack trace: #0 .../wp-includes/template.php(730): require_once() #1 .../wp-includes/template.php(676): load_template('.../...', true, Array) #2 .../wp-includes/general-template.php(48): locate_template(Array, true, true, Array) #3 .../wp-content/themes/dotcom/seedlet/page.php(11): get_header() #4 .../wp-includes/template-loader.php(106): include('.../...') #5 .../wp-blog-header.php(19): require_once('.../...') #6 .../index.php(17): require('.../...') #7 ../server.php(191): require('.../...') #8 {main} thrown in
.../wp-content/themes/dotcom/spearhead/header.php
on line
106.../
```

It appears that the `spearhead_cart_link` function defined here is not defined: 

https://github.com/Automattic/themes/blob/ab448ae5e45e87326259f5747cb4c590fc39ffe3/spearhead/header.php#L106

From a quick search through the theme's code, I think that `spearhead_cart_widget` is also not defined at the moment. 

https://github.com/Automattic/themes/blob/ab448ae5e45e87326259f5747cb4c590fc39ffe3/spearhead/header.php#L108

**Comment by kjellr** [comment]
Swapping that out to `seedlet_cart_link()` and `seedlet_cart_widget()` fixes the fatal, but causes a bunch of weird appearance errors: 

Desktop:
<img width="1353" alt="Screen Shot 2021-01-11 at 1 58 42 PM" src="https://user-images.githubusercontent.com/1202812/104226066-3b3f9080-5415-11eb-8a7d-fb57dec1cc23.png">

Mobile: 

Closed|Open
---|---
<img width="336" alt="Screen Shot 2021-01-11 at 1 58 50 PM" src="https://user-images.githubusercontent.com/1202812/104226090-4397cb80-5415-11eb-8db2-bfc4aca48a5d.png">|<img width="335" alt="Screen Shot 2021-01-11 at 1 58 57 PM" src="https://user-images.githubusercontent.com/1202812/104226091-4397cb80-5415-11eb-8c7b-75767e90902a.png">

Related: #2715

**Comment by scruffian** [comment]
Fixed by #2745
